asapdiscovery-alchemy/asapdiscovery/alchemy/cli.py

```diff
@@ -278,5 +278,35 @@ def status(network: str):
     client.network_status(planned_network=planned_network)
 
 
+@cli.command()
+@click.option(
+    "-n",
+    "--network",
+    type=click.Path(resolve_path=True, readable=True, file_okay=True, dir_okay=False),
+    help="The name of the JSON file containing a planned FEC network.",
+    default="planned_network.json",
+    show_default=True,
+)
+@click.option(
+    "-v",
+    "--verbose",
+    count=True,
+    help="Verbosity of output; one `-v` will give ScopedKeys of restarted Tasks",
```

Comment:
would multiple `v`s do anything here? If not, should we add multiple levels?

Reply:
Realized that if we're only using one count, kinda silly to make this count-based. Switched it to a simple flag.